### PR TITLE
fixed duplicate entries issue

### PR DESCRIPTION
**Pull request by viditchopra1500**
Now if somebody adds a song already in the list then an alert dialog box will appear with the message "the song is already in the list"

### CODE REVIEW

**Review by abhisheknaiidu — CHANGES_REQUESTED** [review]
Good, but one suggestion why not we show the dialog box, just after clicking the `ADD` rather than clicking on `ADD SONG` , It will make better User Experience.

**Comment by viditchopra1500** [comment]
Now the dialog box appears after clicking ADD

File: src/components/AddSong.js [hunk 1]
```diff
@@ -226,4 +238,4 @@ async function handleAddSong() {
 }
 
 export default AddSong;
- 
\ No newline at end of file
+
```
Comment (abhisheknaiidu) [inline src/components/AddSong.js]: Remove this extra `line-space`

File: src/components/AddSong.js [hunk 2]
```diff
@@ -136,6 +137,17 @@ async function handleAddSong() {
         showDialog(false)
     }
 
+    function handleShowDialogBox(){
+        var flag=0
```
Comment (abhisheknaiidu) [inline src/components/AddSong.js]: Change `var` to es6 variables `const` or `let`

**Review by abhisheknaiidu — CHANGES_REQUESTED** [review]
Looks Great, but could you use this package for alerts, it will improve the interface greatly.

https://github.com/t4t5/sweetalert

File: package.json [hunk 3]
```diff
@@ -19,6 +19,7 @@
     "react-player": "^2.3.1",
     "react-scripts": "3.4.1",
     "subscriptions-transport-ws": "^0.9.16"
```
Comment (abhisheknaiidu) [inline package.json]: Add `,` in the last, it's giving error

File: src/components/AddSong.js [hunk 4]
```diff
@@ -136,6 +138,16 @@ async function handleAddSong() {
...
     }
 
+    function handleShowDialogBox(){
+        let flag=0
+        data.songs.forEach(element => {
+            if(element.url===url){
+            flag=1
+            }
+        });
+        flag===1?swal('Song already present in list!!'):showDialog(true)
```
Comment (abhisheknaiidu) [inline src/components/AddSong.js]: change it to `swal("Oops" ,  "Song is already present in list!!" ,  "error")`

**Review by abhisheknaiidu — CHANGES_REQUESTED** [review]
I tried to make small changes and commit in you're PR, but not able to make it, because of `forked` repo maybe

**Review by abhisheknaiidu — APPROVED** [review]
Great!

**Pull request merged**